arangod/Scheduler/SocketTask.cpp

```diff
@@ -50,14 +50,14 @@ SocketTask::SocketTask(arangodb::EventLoop loop,
                        arangodb::ConnectionInfo&& connectionInfo,
                        double keepAliveTimeout, bool skipInit = false)
     : Task(loop, "SocketTask"),
-      _connectionStatistics(nullptr),
+      _peer(std::move(socket)),
       _connectionInfo(std::move(connectionInfo)),
+      _connectionStatistics(nullptr),
```

Comment:
I have seen changes to statistics in various places in this PR. Where is the statistics code, does it need to be ported back as well?

Reply:
I think this mostly stayed the same, originally (in devel) I just simplified some code